services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -96,8 +96,14 @@ async def listen(app: web.Application, db_engine: Engine):
         await conn.execute(listen_query)
 
         while True:
-            # NOTE: this waits for a new notification so the engine is locked here
-            notification = await conn.connection.notifies.get()
+            # NOTE: instead of using await get() we check first if the connection was closed
+            # since aiopg does not reset the await in such a case (if DB was restarted or so)
+            if conn.closed:
+                raise ConnectionError("connection with database is closed!")
+            if conn.connection.notifies.empty():
+                await asyncio.sleep(1)
```

Comment:
Could you explain why you are using 1 second here?

Reply:
well. no.
I created a constant instead of just a value. before it would return right after a new message would come in the queue. I don't think it is super critical but still is nicer that the user does not wait too long to get an update.
so for now I find it to be a compromise. if it's too slow we can still reduce it. Anyway once the fix comes in, we might change it again.